easybuild/easyconfigs/h/HPCBIOS_Bioinfo/HPCBIOS_Bioinfo-20130829-ictce-5.3.0.eb

```diff
@@ -56,7 +56,7 @@ dependencies = [
     ('mpiBLAST', '1.6.0'),
     ('NEURON', '7.2'),
     ('Oases', '0.2.08'),
-    ('PAML', '4.7'),
+    ('PAML', '4.7a'),
```

Comment:
I guess you have an interesting reason for your diff, yet I'm inclined to ask what happened to this!